js/views/composer.js

```diff
@@ -509,6 +509,11 @@ define(function(require) {
 					alias =  _.find(this.aliases, function(alias) {
 						return alias.emailAddress  === require('state').currentAccount.get('email');
 					});
+				} else {
+					var firstAccount = this.accounts.at(1);
```

Comment:
if you add one account and then another, the accounts are ususally ordered as account1, unified account, account2. Therefore you'd select the unified inbox, although you actually want the first account. Use something like ``this.accounts.filter(…accountId not -1…).at(0)``.